Src/WindowsServer/WindowsServer.Nuget/Package.nuspec

```diff
@@ -22,6 +22,12 @@
         <dependency id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="[$coresdkversion$]" />
         <dependency id="Microsoft.ApplicationInsights.PerfCounterCollector" version="$version$" />
       </group>
+      <group targetFramework="netstandard1.6">
```

Comment:
Please create a Github task to validate that the new assembly for netstandard is also signed. (We previously had an issue when DepCollector was added for netcore, the assembly was not signed)

Reply:
Can I simply verify this myself, now? Or do we need  a task to do this?